vscode-dotty/src/worksheet.ts

```diff
@@ -0,0 +1,346 @@
+'use strict'
```

Comment:
This isn't actually necessary with recent versions of typescript (it's automatically inserted in the generated js)